duckduckgo/iOS
# iOS11 Disconnect Content Blocking

Reviewer: Chris
Asana: https://app.asana.com/0/414235014887631/449517770398665

**Description**:
This PR includes two components
- Apple content blocking for the disconnect list
- Native tracker detection

Whitelisting and associated domain check will be done as part of a later task

**Steps to test this PR**:

TEST 1:
Go to a website e.g evans cycles with an iOS 11 device and ensure that disconnect trackers are showing as blocked. Also check the safari console to ensure that they are blocked there

TEST 2:
Toggle content blocking on and off and repeat the above steps to ensure that trackers are only blocked when it is on

TEST 3:
Go to a website e.g evans cycles with and iOS 10 device and ensure that content blocking still works as expected

###### Reviewer Checklist:


###### PR DRI Checklist:
